packages/locale/lang/sw.ts

```diff
@@ -0,0 +1,165 @@
+export default {
+  name: 'en',
+  el: {
+    colorpicker: {
+      confirm: 'sawa',
+      clear: 'futa',
+      defaultLabel: 'kichagua rangi',
+      description:
```

Comment:
The keyword surrounded by `{}` is the keyword for replacing. Please don't translate it. The same below.